esp/src/src-react/components/SetLogicalFileAttr.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from "react";
+import { DefaultButton, DetailsList, DetailsListLayoutMode, IColumn } from "@fluentui/react";
+import { SizeMe } from "react-sizeme";
+import { csvParse } from "d3-dsv";
+import { DaliService, WsDali } from "@hpcc-js/comms";
+import { scopedLogger } from "@hpcc-js/util";
+import { TableGroup } from "./forms/Groups";
+import nlsHPCC from "src/nlsHPCC";
+import { HolyGrail } from "../layouts/HolyGrail";
+
+class MyDaliService extends DaliService {
+    SetLogicalFileAttr(request: WsDali.SetLogicalFilePartAttrRequest): Promise<WsDali.ResultResponse> {
+        return this._connection.send("SetLogicalFileAttr", request, "json", false, undefined, "ResultResponse");
+    }
+}
+
+const logger = scopedLogger("src-react/components/SetLogicalFileAttr.tsx");
+
+const myDaliService = new MyDaliService({ baseUrl: "" });
```

Comment:
If you did remove the extended class definition above, this `new MyDaliService(...` would just be `new DaliService(...`. And this would apply to each of the components below as well.